charts/dendrite/values.yaml

```diff
@@ -0,0 +1,599 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default
+# values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+---
+image:
+  # -- image repository
+  repository: ghcr.io/matrix-org/dendrite-monolith
+  # -- image tag
+  # @default -- chart.appVersion
+  tag:
+  # -- image pull policy
+  pullPolicy: IfNotPresent
```

Comment:
missing `PullSecrets` for own mirrors and own image registries

Reply:
> missing `PullSecrets` for own mirrors and own image registries

Nope, https://github.com/k8s-at-home/library-charts/blob/main/charts/stable/common/values.yaml#L49